crypto/rand/drbg_lib.c

```diff
@@ -33,9 +28,79 @@ static const char ossl_pers_string[] = "OpenSSL NIST SP 800-90A DRBG";
  * a much bigger deal than just re-setting an allocated resource.)
  */
 
+/*
+ * The three shared DRBGs
+ *
+ * There are three shared DRBGs (master, public and private), which are
+ * accessed concurrently by all threads.
+ *
+ * The master DRBG
+ *
+ * Not used directly by the application, only for reseeding the two other
+ * DRBGs. It reseeds itself by pulling either randomness from os entropy
+ * sources or by consuming randomnes which was added by RAND_add()
+ */
+static RAND_DRBG drbg_master;
+/*
+ * The public DRBG
+ *
+ * Used by default for generating random bytes using RAND_bytes().
```

Comment:
Maybe you should add a note somewhere that we use it internally for numbers that become public, like in the RAND_bytes man page, plus expand on the history.